Backend/NotificationService/tests/test_notificationmanager.cpp

```
@@ -1,223 +1,128 @@
+    void deleteConnection(const SocketPtr &socket) override {
+
+    }
+    void saveConnections(UserId, SocketPtr socket) override {
+
+    }
+    SocketPtr getUserSocket(UserId) override {
+
+    }
+    bool userOnline(UserId) override {
+
+    }
```

Cppcheck (2.19.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)